freshmaker/image_verifier.py

```diff
@@ -180,15 +153,17 @@ def verify_repository(self, repo_name):
 
         data = {
             "repository": {"auto_rebuild_tags": repo["auto_rebuild_tags"]},
-            "images": {}
+            "images": {},
         }
 
-        images = self.lb.find_images_with_included_rpms(
-            [], [], {repo["repository"]: repo}, include_rpm_manifest=False)
+        images = self.pgql.find_images_by_installed_rpms(
```

Comment:
While testing locally, I noticed this API is implemented in a way that it needs to have the argument `rpm_names` with some value, otherwise this will always be false `query_filter["and"].append({"rpm_manifest": {"rpms": {"name": {"in": rpm_names}}}})`. However, I also noticed that lb API `find_images_with_included_rpms` will be working fine, even if `rpm_nvrs` is None for example, because the `if` statement (https://github.com/redhat-exd-rebuilds/freshmaker/blob/master/freshmaker/lightblue.py#L996) will not pass so the api will just ignore that `rpm_nvrs` is None and more further. So I'm not sure if this is intended behaviour of the PyxisGQL api? @qixiang